src/middlewared/middlewared/plugins/pool.py

```diff
@@ -245,9 +250,9 @@ def configure_resilver_priority(self):
         sysctl.filter('vfs.zfs.resilver_min_time_ms')[0].value = resilver_min_time_ms
         sysctl.filter('vfs.zfs.scan_idle')[0].value = scan_idle
 
-    @accepts(Str('volume'), Str('fs_type'), Str('dst_path'))
-    @job(lock=lambda args: 'volume_import')
-    async def import_disk(self, job, volume, fs_type, dst_path):
+    @accepts(Str('volume'), Str('fs_type'), Str('msdosfs_locale'), Str('dst_path'))
+    @job(lock=lambda args: 'volume_import', logs=True)
+    async def import_disk(self, job, volume, fs_type, msdosfs_locale, dst_path):
```

Comment:
@themylogin  A new argument for every different filesystem seems to easily break backward-compatibility.

Do you think it would be a bad idea to have a `fs_options`, instead? So say in the future we could have a few others for NTFS/UFS without breaking the API?